desertbot/modules/commands/Sed.py

```diff
@@ -158,12 +170,14 @@ def storeMessage(self, message, unmodified=True):
         if message.replyTo not in self.messages:
             self.messages[message.replyTo] = []
             self.unmodifiedMessages[message.replyTo] = []
-        self.messages[message.replyTo].append(message)
-        self.messages[message.replyTo] = self.messages[message.replyTo][-self.historySize:]
+        userList = self.unmodifiedMessages[message.replyTo]
+        userList.append(message)
+        self.messages[message.replyTo] = userList[-self.historySize:]
 
         if unmodified:
-            self.unmodifiedMessages[message.replyTo].append(message)
-            self.unmodifiedMessages[message.replyTo] = self.unmodifiedMessages[message.replyTo][-self.historySize:]
+            userList = self.unmodifiedMessages[message.replyTo]
+            userList.append(message)
+            self.unmodifiedMessagesi[message.replyTo] = userList[-self.historySize:]
```

Comment:
Should be self.unmodifiedMessages[message.replyTo]

Reply:
There's also some stuff broken with DataStore but that's unrelated to this, I'll fix it in master.